### PR TITLE
Viktor Svensson React-portfolio

### PR DESCRIPTION
**Pull request by ViktorSvenssonN**
https://glittering-yeot-152860.netlify.app/

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,30 @@
...
+/* Component that prints all the importet compnents */
 
 export const App = () => {
   return (
     <div>
-      Find me in src/app.js!
+      <GlobalFonts />
+      <Header />
+      <AboutMe />
+      <Tech color="#f0a04b" />
```
Comment (Sofia-Andersson) [inline src/App.js:21]: Good way of solving the colour-theme in a way that can easily be overviewed and changed.

File: src/components/AboutMe.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+              I’ve always been about making peoples days a little better whether it’s behind a cashier or in a support call, doing it with awesome applications and websites is even better.
+            </AboutMeParagraph>
+          </AboutMeBox>
+        </AboutMeContainer>
+      </InnerWrapper>
+    </OuterWrapper>
+  );
+};
+
+/* Styled components for About Me */
```
Comment (Sofia-Andersson) [inline src/components/AboutMe.js:26]: Good work with comments clearly explaining whats happening, not only here but throughout your whole project 👍

File: src/components/AboutMe.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    align-items: center;
+    
+`;
+
+const AboutMeParagraph = styled.p`
+    color: #183a1d;
+    font-size: 24px;
+    line-height: 1.8;
+  
+    &:hover {
```
Comment (Sofia-Andersson) [inline src/components/AboutMe.js:49]: I like your consistent use of hover effects through the whole project, they are discrete yet makes the page feel more alive. Which I would say gives a good user experience. :)

File: src/components/GlobalStyleComponents.js [hunk 4]
```diff
@@ -0,0 +1,199 @@
+import styled, { createGlobalStyle } from 'styled-components';
```
Comment (Sofia-Andersson) [inline src/components/GlobalStyleComponents.js:1]: Very good and structured solution for the Global Styling! :)

File: src/components/MyThoughts.js [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+
+  @media (max-width: 800px) {
+  font-size: 16px;
+  line-height: 1.4;
+}
+
+`;
+
+/* const Wrapper = styled.div`
+  width: 80%;
```
Comment (Sofia-Andersson) [inline src/components/MyThoughts.js:76]: Has this part maybe been moved to the Global Styling component? And can be removed here?

**Review by Sofia-Andersson — COMMENTED** [review]
Thanks for giving me the opportunity to review a very good and well made example of a React-portfolio! 😊👍 Which doubtless meets this weeks requirements.
I will definitely bring with me some inspiration for future steps to my own portfolio.

I like your structured and consistent way of writing comments - makes understanding and reviewing much easier, and a very good thing for future work-roles of course.

I like your way of using hover-effects, in most sections and in a tasteful way: discrete but still impactful. I am also impressed by your way of creating such "clean" and still great working media-queries. Adding the color-palett to your project shows your structured way of working.

In summary I think you should be proud of your portfolio and I think future employers will also appreciate it. Great work last week! 😊
